Argo/Functions/catDecoded.swift

```diff
@@ -1,11 +1,26 @@
 public func catDecoded<T>(xs: [Decoded<T>]) -> [T] {
-  return xs.reduce([]) { accum, elem in
-    elem.map { accum + [$0] } ?? accum
+  var accum: [T] = []
+  accum.reserveCapacity(xs.count)
+  
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)